doc/source/admin/migrating_to_gunicorn.md

```diff
@@ -0,0 +1,45 @@
+# Migrating from uWSGI to Gunicorn and FastAPI
+
+## Why do we migrate away from uWSGI
+
+Starting with Galaxy release 22.01 the default application webserver is [Gunicorn](https://gunicorn.org/), which replaces [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/). We will remove support for running Galaxy under uWSGI in Galaxy release 22.05.
+We have made this move because uWSGI support for newer Python versions has been problematic and because
+we have modernized the API portion of our backend. The Galaxy web application is now an ASGI application
+driven by [FastAPI](https://fastapi.tiangolo.com/) and [Starlette](https://www.starlette.io/).
+This will enable numerous important features in the future that would not be possible (or would be more complicated to set up)
+otherwise. Importantly, [FastAPI's documentation](https://fastapi.tiangolo.com/tutorial/) is very good and
+should increase developer productivity.
+
+The main advantage for our users today is that Galaxy hosts documentation for its API as an [OpenAPI](https://www.openapis.org/)
+document that is directly generated from Galaxy's source code, and so will never be out of date or differ
+from the actual implementation.
+Note that not all API endpoints have been modernized as of yet, this is an ongoing process.
+
+To see the API documentation for your Galaxy server you can go to `<galaxy_url>/api/docs`.
```

Comment:
```suggestion
To see the API documentation for your Galaxy server you can go to `<galaxy_url>/api/docs`.
```

This doesn't seem to work on https://usegalaxy.org/api/docs . Needs to be configured somewhere? Proxy?

Reply:
usegalaxy isn't running Gunicorn yet